tests/mocks/handler.ts

```diff
@@ -35,6 +35,7 @@ export type HandlerConstructorConfig = {
   autoStorage?: boolean;
   cacheRefreshCycles?: number;
   runtimeRpcs?: string[];
+  rpcTimeout?: number;
```

Comment:
I think all these optional properties are bad idea. Instead I think it's more robust to check for default values that indicate false or null instead. 

In my experience it makes dealing with the object more unwieldy. But there's certainly situations where it can be more convenient to implement like this.